app/bundles/CoreBundle/Twig/Helper/EntityHelper.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Mautic\CoreBundle\Twig\Helper;
+
+use Doctrine\ORM\EntityManagerInterface;
+use Twig\Extension\AbstractExtension;
+use Twig\TwigFunction;
+
+class EntityHelper extends AbstractExtension
+{
+    private EntityManagerInterface $entityManager;
+
+    public function __construct(EntityManagerInterface $entityManager)
+    {
+        $this->entityManager = $entityManager;
+    }
+
+    /**
+     * Registers the custom Twig functions.
+     */
+    public function getFunctions(): array
+    {
+        return [
+            new TwigFunction('getEntities', [$this, 'getEntities']),
+        ];
+    }
+
+    /**
+     * Retrieves entities based on the provided class name and ID(s).
+     *
+     * @param string           $entityName the fully qualified class name of the entity
+     * @param int|string|array $ids        a single ID or an array of IDs to retrieve
+     *
+     * @return array the array of retrieved entities
+     */
+    public function getEntities(string $entityName, int|string|array $ids): array
```

Comment:
Please require just an array. This multiple type usage is an anti-pattern. If there is just 1 entity needed and you don't want to call it as `getEntities('page', [2])` then create a new method called getEntity that will accept string ID. It can internally call getEntities and returning just the first result.